package.json

```diff
@@ -14,6 +14,8 @@
     "@auth0/auth0-react": "^2.0.2",
     "@headlessui/react": "1.7.11",
     "@heroicons/react": "^2.0.10",
+    "@react-hook/size": "^2.1.2",
+    "@react-hook/window-size": "^3.1.1",
```

Comment:
Do you still need these? I don’t see them imported anywhere. Not sure if they’re an internal dependency of react-svg-pan-zoom.

Reply:
oh I don't, I'll remove them